app/containers/Forms/NumberFields.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import NumberFormat from 'react-number-format';
+import {FieldProps} from 'react-jsonschema-form';
+
+const NumberFields: React.FunctionComponent<FieldProps> = ({
```

Comment:
Rename this (and the file to `NumberField`)

Reply:
Renamed